### DISCUSSION TITLE
How i can install cockpit in vm

### DISCUSSION THREAD

**Discussion by Prince2319** [cockpit-project/cockpit#18414] [Q&A] [answered]
![Screenshot from 2023-02-27 15-53-45](https://user-images.githubusercontent.com/123302204/221566918-fb60b4a2-b33a-4182-aa29-9776074466c7.png)
I have done till here now what should i do

**Comment by garrett** [comment, marked as answer]
I think you're asking to be able to install and run VMs in Cockpit. You need to install the `cockpit-machines` package on your system. Then you'll see it in the sidebar, as "Virtual machines". It'll look like this:

![image](https://user-images.githubusercontent.com/10246/221815656-2bc26af8-5d2f-4a9b-81fa-05274204f62b.png)

(Additionally, you also need to have virtual machines enabled in your BIOS / EFI Settings. It's that thing you can enter on boot when you hit some key like Del, ESC, F12, F11, F2... _most_ machines tell you the key during boot. But this is true for running VMs on any machine. Some computers enable it by default and some have it disabled by default. If you can already run VMs, then it's probably set correctly.)

**Reply by Prince2319** [reply]
Yes SIr, now how can i add podman containers and navigators can you tell me the commands .
Please!

**Reply by KKoukiou** [reply]
You can see the package names of all available applications here https://cockpit-project.org/applications 
For navigator check https://github.com/45Drives/cockpit-navigator/

**Reply by Prince2319** [reply]
Thanks Mam
